### PR TITLE
doc: update README.md

### PR DESCRIPTION
**Pull request by mantissahz**
#### Which issue(s) this PR fixes:

Issue # longhorn/longhorn#6914

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,69 +6,89 @@ Manager for Longhorn.
...
-`kubectl create -Rf deploy/install`
+- `kubectl create -f https://raw.githubusercontent.com/longhorn/longhorn/master/deploy/longhorn.yaml`
 
 It will deploy the following components in the `longhorn-system` namespace:
-1. Longhorn Manager
-2. Longhorn Flexvolume Driver for Kubernetes
-3. Longhorn UI
+
+- Longhorn Managers
+- Longhorn Flexvolume Driver for Kubernetes
```
Comment (derekbit) [inline README.md:30]: `Longhorn Flexvolume Driver for Kubernetes` is not supported, right?

File: README.md [hunk 2]
```diff
@@ -6,69 +6,89 @@ Manager for Longhorn.
...
 
-`kubectl create -Rf deploy/install`
+- `kubectl create -f https://raw.githubusercontent.com/longhorn/longhorn/master/deploy/longhorn.yaml`
 
 It will deploy the following components in the `longhorn-system` namespace:
-1. Longhorn Manager
-2. Longhorn Flexvolume Driver for Kubernetes
-3. Longhorn UI
+
+- Longhorn Managers
```
Comment (derekbit) [inline README.md:29]: Add Longhorn CSI plugin components

File: README.md [hunk 3]
```diff
@@ -6,69 +6,89 @@ Manager for Longhorn.
...
-2. Make sure `iscsiadm`/`open-iscsi` has been installed on the host.
+- Existing Kubernetes Cluster 1.21+.
+- [Mount propagation](https://kubernetes-csi.github.io/docs/deploying.html#enabling-mount-propagation) must be enabled.
+- Make sure `iscsiadm`/`open-iscsi` and `nfs-common`/`nfs-utils`/`nfs-client` has been installed on the host.
+  - Recommend that you execute the [environment_check.sh](https://raw.githubusercontent.com/longhorn/longhorn/master/scripts/environment_check.sh) script to check if all requirements have been fulfilled.
+- The host filesystem supports the `file extents` feature to store the data. Currently we support:
+  - ext4
+  - XFS
+
+For more details of the installation requirements, please refer to the [Longhorn documentation.](https://longhorn.io/docs/latest/deploy/install/#installation-requirements)
```
Comment (jillian-maroket) [inline README.md:17]:
```suggestion
- Kubernetes cluster: 
  - Kubernetes v1.21 or later is installed.
  - [Mount propagation](https://kubernetes-csi.github.io/docs/deploying.html#enabling-mount-propagation) is enabled.
- Host:
  - `iscsiadm`/`open-iscsi` and `nfs-common`/`nfs-utils`/`nfs-client` are installed.
  - The filesystem (ext4 or XFS) supports the `file extents` feature for data storage.

Run the [environment_check.sh](https://raw.githubusercontent.com/longhorn/longhorn/master/scripts/environment_check.sh) script to check if your system meets the listed requirements. For more information, see the [Longhorn documentation](https://longhorn.io/docs/latest/deploy/install/#installation-requirements).
```

File: README.md [hunk 4]
```diff
@@ -6,69 +6,89 @@ Manager for Longhorn.
...
 It will deploy the following components in the `longhorn-system` namespace:
-1. Longhorn Manager
-2. Longhorn Flexvolume Driver for Kubernetes
-3. Longhorn UI
+
+- Longhorn Managers
+- Longhorn Flexvolume Driver for Kubernetes
+- Longhorn UI
+
+For more details of the installation requirements, please refer to the [Longhorn documentation.](https://longhorn.io/docs/latest/deploy/install/)
```
Comment (jillian-maroket) [inline README.md:33]:
```suggestion
- Longhorn Manager
- Longhorn FlexVolume Driver for Kubernetes
- Longhorn UI

For more information, see the [Longhorn documentation](https://longhorn.io/docs/latest/deploy/install/).
```

File: README.md [hunk 5]
```diff
@@ -6,69 +6,89 @@ Manager for Longhorn.
...
 Longhorn CRD has finalizers in them, so user should delete the volumes and related resource first, give manager a chance to clean up after them.
 
 To prevent damage to the Kubernetes cluster, we recommend deleting all Kubernetes workloads using Longhorn volumes (PersistentVolume, PersistentVolumeClaim, StorageClass, Deployment, StatefulSet, DaemonSet, etc).
 
-1. Create the uninstallation job to cleanly purge CRDs from the system and wait for success:
-  ```
-  kubectl create -f deploy/uninstall/uninstall.yaml
-  kubectl get job/longhorn-uninstall -w
-  ```
+1. Make sure the value of the Longhorn setting `deleting-confirmation-flag` is `true`.
```
Comment (jillian-maroket) [inline README.md:41]:
```suggestion
1. Ensure that the value of the Longhorn setting `deleting-confirmation-flag` is `true`.
```

File: README.md [hunk 6]
```diff
@@ -6,69 +6,89 @@ Manager for Longhorn.
...
 
-Example output:
+```bash
+kubectl -n longhorn-system edit setting deleting-confirmation-flag
 ```
-$ kubectl create -f https://raw.githubusercontent.com/rancher/longhorn/master/uninstall/uninstall.yaml
+
+2. Create the uninstallation job to cleanly purge CRDs from the system and wait for success:
```
Comment (jillian-maroket) [inline README.md:47]:
```suggestion
1. Create the uninstallation job to cleanly purge CRDs from the system. Verify that uninstallation was successful.
```

File: README.md [hunk 7]
```diff
@@ -6,69 +6,89 @@ Manager for Longhorn.
...
 longhorn-uninstall   1/1           20s        20s
 ^C
 ```
 
-2. Remove remaining components:
-  ```
-  kubectl delete -Rf deploy/install
-  kubectl delete -f deploy/uninstall/uninstall.yaml
-  ```
+3. Remove remaining components:
```
Comment (jillian-maroket) [inline README.md:70]:
```suggestion
1. Remove the remaining components.
```

**Review by jillian-maroket — COMMENTED** [review]
@mantissahz Let me know if you have concerns about the changes.

File: README.md (hunk 1 above)
Comment (mantissahz) [reply README.md]: Removed. Thanks.

File: README.md (hunk 2 above)
Comment (mantissahz) [reply README.md:29]: Added. Thanks

File: README.md [hunk 8]
```diff
@@ -6,69 +6,89 @@ Manager for Longhorn.
 
 ## Requirement
```
[the remark this answers is not in the document]
Comment (mantissahz) [reply README.md]: Modified. Thanks

File: README.md (hunk 3 above)
Comment (mantissahz) [reply README.md]: Modified. Thanks, it's more clear.

File: README.md (hunk 6 above)
Comment (mantissahz) [reply README.md]: Modified. Thanks

File: README.md (hunk 5 above)
Comment (mantissahz) [reply README.md]: Modified. Thanks

File: README.md (hunk 4 above)
Comment (mantissahz) [reply README.md]: Modified. Thanks

File: README.md (hunk 7 above)
Comment (mantissahz) [reply README.md]: Modified. Thanks

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

File: README.md [hunk 9]
```diff
@@ -4,71 +4,90 @@ Longhorn Manager
...
 
-`kubectl create -Rf deploy/install`
+- `kubectl create -f https://raw.githubusercontent.com/longhorn/longhorn/master/deploy/longhorn.yaml`
 
 It will deploy the following components in the `longhorn-system` namespace:
-1. Longhorn Manager
-2. Longhorn Flexvolume Driver for Kubernetes
-3. Longhorn UI
+
+- Longhorn Manager
```
[the remark this answers is not in the document]
Comment (mantissahz) [reply README.md:28]: Added. Thanks.

File: README.md [hunk 10]
```diff
@@ -4,71 +4,91 @@ Longhorn Manager
 
 Manager for Longhorn.
 
-## Requirement
+## Requirements
 
-1. Existing Kubernetes Cluster 1.16+.
-2. Make sure `iscsiadm`/`open-iscsi` has been installed on the host.
+- Kubernetes cluster:
```
Comment (innobead) [inline README.md:9]: nit: The other way is to refer to the latest doc instead of duplicating the requirements here again.

**Review by innobead — APPROVED** [review]
LGTM.

**Pull request merged**